RELEASE_NOTES.md

```diff
@@ -16,8 +16,8 @@ https://github.com/scalyr/scalyr-agent-2/blob/master/CHANGELOG.md.
   seamless for the majority of cases, but if you use custom monitors that require additional libraries, you will need to
   specify those requirements in new agent's configuration.
   
-  After upgrading to a new version, add required libraries to the file `/opt/scalyr-agent-2-dependencies/etc/additional-requirements.txt` 
-  and run command `/opt/scalyr-agent-2-dependencies/bin/agent-libs-config initialize` to re-initialize requirements.
+  After upgrading to a new version, add required libraries to the file `/opt/scalyr-agent-2/etc/additional-requirements.txt` 
```

Comment:
These are developer notes right?  Maybe it would be clearer to say "When creating a new version"

Reply:
That's serves more like an additional reference where user or developer can find more info about some particular change if it's big enough to be in the CHANGELOG itself. That's because, (as I can see it), the CHANGELOG has to follow is special format where each "change" must not have a line breaks, so a really long text would be ugly and very inconvenient to edit, so it's just better to move it. to another file.

And no, that's more a user note, about how to install user's custom agent dependencies after the upgrade to a new version.